packages/civic-auth-web3/solana/next14-wallet-adapter/src/app/SendTransaction.tsx

```diff
@@ -0,0 +1,114 @@
+"use client";
+
+import { useConnection } from "@solana/wallet-adapter-react";
+import { useUser } from "@civic/auth-web3/react";
+import { userHasWallet, Web3UserContextType } from "@civic/auth-web3";
+
+import {
+  Connection,
+  LAMPORTS_PER_SOL,
+  PublicKey,
+  SystemProgram,
+  Transaction,
+} from "@solana/web3.js";
+import { useCallback, useState } from "react";
+
+// Separate component for the app content that needs access to hooks
+const SendTransaction = () => {
+  const userContext = useUser();
+  const wallet = userHasWallet(userContext)
+    ? userContext.solana.wallet
+    : undefined;
+  const { connection } = useConnection();
+
+  const [recipientAddress, setReceipientAddress] = useState<string>(
+    "6GyYxtn4tQPgGS3WhZeeLeC6TqsNUSKqfZLH1MUhAoGe"
```

Comment:
should we hardcode an address in here? maybe better to make it an env var?

Reply:
Yes that's an oversight, will fix

Reply:
fixed